### PR TITLE
Bindings for MaskedViewIOS

### PR DESCRIPTION
**Pull request by grabbou**
Implements bindings for MaskedViewIOS - used in `rebolt-navigation` right now

### CODE REVIEW

**Comment by grabbou** [comment]
@souhe @czystyl let me know what you thinkit!

File: src/components/maskedViewIOS.re [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      ~needsOffscreenAlphaCompositing=?,
+      ~renderToHardwareTextureAndroid=?,
+      ~accessibilityViewIsModal=?,
+      ~shouldRasterizeIOS=?,
+    ) =>
+  ReasonReact.wrapJsForReason(
+    ~reactClass=view,
+    ~props=
+      Props.extendView(
+        ~accessibilityLabel?,
```
Comment (czystyl) [inline src/components/maskedViewIOS.re]: Can't we omit that duplication in some way? or we need and my question is dump

File: docs/masked-view-ios.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: MaskedViewIOS (x)
+title: MaskedViewIOS
 ---
 
 ## Example of use
```
Comment (souhe) [inline docs/masked-view-ios.md]: It would be nice to have docs for this component ;)

File: src/components/maskedViewIOS.re (hunk 1 above)
Comment (grabbou) [reply src/components/maskedViewIOS.re]: Unfortunately, there's no way (I know you are thinking of spread operator). Anyway, this is something that I actually like about Reason - see, you put `Props.extendView` call and immediately type checker will let you know you are missing some properties.

Comment (czystyl) [reply src/components/maskedViewIOS.re]: Sure that's great!

**Pull request merged**